### PR TITLE
828: revert link styling for button and add new kind "invisible"

### PR DESCRIPTION
**Pull request by chrisdopuch**
## PR Checklist

- Manually tested across supported browsers
  - [X] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

### CODE REVIEW

**Comment by chrisdopuch** [comment]
Still need to hear back from @rsadres on what components should use this kind of button

**Comment by chrisdopuch** [comment]
Demo: http://docspot.devnxs.net/projects/lucid/828/

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lucid-ui",
-  "version": "2.21.0",
+  "version": "2.21.1-0",
```
Comment (ogupte) [inline package.json]: should this version change be merged to master?

File: src/components/Button/examples/invisible.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	flexDirection: 'column',
+};
+
+const articleStyle = {
+	display: 'flex',
+	margin: '5px 0',
+};
+
+const buttonStyle = {
+	marginRight: '5px',
```
Comment (ogupte) [inline src/components/Button/examples/invisible.jsx]: `marginRight: 5`

File: src/components/Button/examples/link.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ const articleStyle = {
 };
 
 const buttonStyle = {
-	'margin-right': '5px',
+	marginRight: '5px',
```
Comment (ogupte) [inline src/components/Button/examples/link.jsx]: `marginRight: 5,`

File: package.json (hunk 1 above)
Comment (chrisdopuch) [reply package.json]: nope thank you I'll remove

**Pull request merged**